templates/README.txt

```diff
@@ -2,3 +2,36 @@ To generate the templates in this directory run `make templates` at the top-leve
 
 Note that generated artifacts, including these templates, are also attached to releases in GitHub.
 
+### Cluster Setup Guide: Using ClusterClass with OpenStack
+1. Source the environment file:
+   ```shell
+   source $REPO_ROOT/templates/env.rc <path-to-clouds.yaml> <$OS_CLOUD>
+   ```
+
+2. Source the OpenStack environment file:
+   ```shell
+   source $REPO_ROOT/templates/openstack_env.rc <path-to-clouds.yaml> <$OS_CLOUD>
+   ```
+
+### Optional: Set Up Tilt Settings
+This step is optional and intended for those using Tilt for creating ClusterClass and clusters.
+1. Update Tilt settings using the script in the `templates` directory:
+   ```shell
+   $REPO_ROOT/templates/update_tilt-settings.sh <path-to-tilt-settings.yaml-file>
+   ```
+
+## Apply Network and Cluster Configurations
+1. Apply Calico CNI:
+   ```shell
+   kubectl apply -f $REPO_ROOT/workspaces/clusterclass-tilt-capo/test/e2e/data/cni/calico.yaml
+   ```
```

Comment:
This should be in the workload cluster AFAIK.

1. Create kind cluster
2. Initialize CAPI/CAPO
3. Create ClusterClass
4. Create (workload) Cluster
5. Get the workload cluster kubeconfig
6. Apply CNI in the workload cluster

After that you should have a healthy cluster (nodes ready)

Reply:
Gotcha!
What I know, you could apply the `cni` before the ClusterClass and it works too.
But I definitely missed the kubeconfig.

Reply:
If you are going to create a Cluster from the ClusterClass and apply the CNI in that Cluster definitely not. I think you are talking about applying the CNI in the management cluster. This is not needed.

For creating a Cluster from a ClusterClass, the ClusterClass must exist first. To apply the CNI in the cluster, the Cluster must exist first.